packages/opentelemetry-instrumentation-bedrock/opentelemetry/instrumentation/bedrock/__init__.py

```diff
@@ -476,6 +589,34 @@ def _set_amazon_span_attributes(span, request_body, response_body):
             )
 
 
+def _create_metrics(meter: Meter):
```

Comment:
There are duplicate code in anthropic and groq, we may need a clean up in future, like creating a common shared lib for the LLM instrumentation. @nirga ^^

Reply:
Yep! I agree
https://github.com/traceloop/openllmetry/issues/1997

Reply:
Will fix with another PR once the code for https://github.com/traceloop/openllmetry/issues/1997 is ready.